packages/core/src/MatchMaker.ts

```diff
@@ -245,17 +245,17 @@ export function defineRoomType<T extends Type<Room>>(
   handlers[name] = registeredHandler;
 
   if (!isDevMode) {
-    cleanupStaleRooms(name);
+    await cleanupStaleRooms(name);
```

Comment:
This could be breaking. Inserting asynchronous code into what is currently a syncrhonous call could break some apps, and changing to async changes the return type, so it is definitely app breaking.